### PR TITLE
Docs for 1.26.2, 1.27.0 and 1.27.1

### PR DESCRIPTION
**Pull request by piyush-jena**
**Issue number:**

Closes # n/a

**Description of changes:**

- Increments metadata for 1.27.1
- Adds GPU variant info
- Adds packages/versions info

Note: 
this is all generated from bottlerocket sources.

Reference PR: #496  

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

**Review by ytsssun — APPROVED** [review]

**Comment by ytsssun** [comment]
The preview of the change looks good to me. Note that we introduced FIPS variants in 1.27.0. You might want to check with @arnaldo2792 to get the documentation updated in the Variants. Not blocking though.

**Review by KCSesh — APPROVED** [review]
Preview LGTM includes 1.26.2, 1.27.0, and 1.27.1 into main.

File: data/settings/1.27.x/aws.toml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+[[docs.ref.config]]
```
Comment (ginglis13) [inline data/settings/1.27.x/aws.toml:1]: Can we document the behavior of this setting that changed via https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/218 in 1.27.x +?

By default for non-FIPS variants, `settings.aws.config` will default to the base64 encoded representation of 

```
[default]
use_fips_endpoint=false
```

For FIPS variants, this setting will default to the base64 encoded string of

```
[default]
use_fips_endpoint=true
```

Comment (ginglis13) [reply data/settings/1.27.x/aws.toml:1]: I won't block on this though - we can follow up with changes for this

Comment (piyush-jena) [reply data/settings/1.27.x/aws.toml:1]: As discussed, these changes will be added in a separate pull request.

**Pull request merged**